pkg/controller/operators/catalog/operator.go

```diff
@@ -311,7 +312,20 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 	sharedIndexInformers = append(sharedIndexInformers, serviceInformer.Informer())
 
 	// Wire Pods
-	podInformer := k8sInformerFactory.Core().V1().Pods()
+	catsrcReq, err := labels.NewRequirement(reconciler.CatalogSourceLabelKey, selection.Exists, nil)
+	if err != nil {
+		return nil, err
+	}
+	catsrcUpdateReq, err := labels.NewRequirement(reconciler.CatalogSourceUpdateKey, selection.Exists, nil)
+	if err != nil {
+		return nil, err
+	}
+	podLabels := labels.NewSelector()
+	podLabels.Add(*catsrcReq)
+	podLabels.Add(*catsrcUpdateReq)
```

Comment:
`Add` returns a new copy of the selector with the requirement added, so these aren't actually modifying `podLabels` (it will match everything).

https://github.com/kubernetes/apimachinery/blob/v0.21.3/pkg/labels/selector.go#L365-L376